compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolResolver.java

```diff
@@ -1500,6 +1500,19 @@ public BType createInvokableType(List<? extends BLangVariable> paramVars,
 
         boolean foundDefaultableParam = false;
         List<String> paramNames = new ArrayList<>();
+        BInvokableType bInvokableType;
```

Comment:
Shall we move this variable decl inside the `if` block and reset the line:1578, I think it's easy to read when variable decl and usage is close to each other.

Reply:
I think it's better to extract a function out of this block.

Reply:
Fixed.